src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -10,6 +10,7 @@
 			:lend-increment-button-version="lendIncrementExpVersion"
 			:image-enhancement-experiment-version="imageEnhancementExperimentVersion"
 			:show-category-description="showCategoryDescription"
+			:show-fundraising-thermometer="showFundraisingThermometer"
```

Comment:
@foochris Please move all of the thermometer experiment set up, graphql query and apollo data along with tracking code into the FeatureHeroLoan component. This will keep it in context and reduce clutter in the top level page. The tracking can move there too seeing as you only fire an event if it's shown now...

Reply:
@mcstover PR updated per your suggestion above.